modules/appeals_api/config/routes.rb

```diff
@@ -111,4 +111,22 @@
       end
     end
   end
+
+  namespace :supplemental_claims, defaults: { format: 'json' } do
+    cpath = '/appeals_api/v2/decision_reviews/supplemental_claims'
```

Comment:
Missing the `v#` namespace inside supplemental claims. It should be `v2` even though this is the first time we've used SC's. We'll just mirror the versions from the old paths for now so that there's less confusion. After we move everything to the new namespaces, we'll then start incrementing their versions separately.

Reply:
Okee!